Cargo.toml

```diff
@@ -6,8 +6,8 @@ resolver = "2"
 members = ["contracts/*"]
 
 [workspace.dependencies]
-soroban-sdk = "21.7.6"
-soroban-token-sdk = "21.7.6"
+soroban-sdk = "22.0.0-rc.3.2"
```

Comment:
Now we have rc.1 in actions and 3.2 here. Is it ok?

Reply:
That was the latest version for stellar CLI as that's a different tool from the SDK

Reply:
https://github.com/stellar/stellar-cli/releases